pipelines/datasets/br_me_siconfi/utils.py

```
@@ -0,0 +1,729 @@
+    )
+    work = Path(tempfile.mkdtemp(prefix="br_me_siconfi_bqseed_"))
+    staging_root = work / "staging"
+    result: dict = {}
+    try:
+        for table in tables:
+            part_uf = _level_of(table) in ("municipio", "uf")
+            wrote = False
+            for year in range(start_year, end_year + 1):
+                at = client.query(
+                    f"SELECT * FROM `{bq_project}.br_me_siconfi.{table}` "
+                    f"WHERE ano = {year}"
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)